dtfs-central-api/src/helpers/get-specific-tfm-facility-values.test.ts

```diff
@@ -72,37 +83,48 @@ describe('getLatestTfmFacilityValues', () => {
 
     it('should throw a NotFoundError', async () => {
       // Act / Assert
-      await expect(getLatestTfmFacilityValues(facilityId, reportPeriod)).rejects.toThrow(
+      await expect(getSpecificTfmFacilityValues(facilityId, reportPeriod)).rejects.toThrow(
         new NotFoundError(`Failed to find a cover end date for the tfm facility with ukef facility id '${facilityId}`),
       );
     });
   });
 
   describe('when the tfm facility is found', () => {
     beforeEach(() => {
-      findOneByUkefFacilityIdSpy.mockResolvedValue({
-        ...aTfmFacility(),
-        facilitySnapshot: {
-          ...aFacility(),
-          ukefFacilityId: facilityId,
-          interestPercentage: 5,
-          dayCountBasis: 365,
-          coverEndDate: today,
-          coverStartDate: today,
-          coverPercentage: 5,
-        },
-      });
+      findOneByUkefFacilityIdSpy.mockResolvedValue(mockFacility);
     });
 
     it('should return a populated object', async () => {
-      const result = await getLatestTfmFacilityValues(facilityId, reportPeriod);
+      const result = await getSpecificTfmFacilityValues(facilityId, reportPeriod);
+
+      const expected = {
+        coverEndDate: convertTimestampToDate(today),
+        coverStartDate: convertTimestampToDate(today),
+        dayCountBasis: 365,
+        interestPercentage: 5,
+        coverPercentage: 5,
+        value: mockFacility.facilitySnapshot.value,
+      };
+
+      expect(result).toEqual(expected);
+    });
+  });
+
+  describe('when the report period is not provided', () => {
```

Comment:
Please can we wrap the test above this in a similar describe block for 'when the report period is provided'?